### DISCUSSION TITLE
参考文献前面方括号内数字和括号有间隔

### DISCUSSION THREAD

**Discussion by ChenyangRan** [sjtug/SJTUThesis#1038] [提问]
当使用参考文献格式如下：
`\usepackage[backend=biber,style=gb7714-2015,gbnamefmt=lowercase,gbpub=false,gbpunctin=false,doi=false,eprint=false,gbalign=center,url=false]{biblatex}` 
编译出的参考文献前代表顺序的方括号内部数字和括号之间有间隔
![1716133148378](https://github.com/sjtug/SJTUThesis/assets/29306735/ccd07878-4566-44bd-b6fa-ff38f9f2926c)
希望在保留该参考文献格式的情况下使得两位数以内的参考文献格式和三位数的一样，如下：
![1716133205424](https://github.com/sjtug/SJTUThesis/assets/29306735/9696a41d-4cd3-4fb1-88f3-38579e8ff32b)

**Comment by LogCreative** [comment]
duplicate #1029
